src/platform/site-wide/announcements/tests/selectors.unit.spec.js

```diff
@@ -106,4 +111,14 @@ describe('selectAnnouncement', () => {
       '"disabled dummy6" preceded "dummy6" but was ignored because of the disabled flag.',
     );
   });
+
+  it('filters our expired announcements', () => {
```

Comment:
I think I meant to say `out` here, but `our` also makes sense in a fun, inclusive way 😆